src/components/home/Banner/Banner.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Button from '../../common/Button/Button';
+import CardImage from '../../common/CardImage/CardImage';
+import Typography from '../../common/Typography/Typography';
+import './Banner.scss';
+
+const Banner = () => {
+  return (
+    <div className="Banner__root w-full">
+      <CardImage alt="Banner" url="/assets/images/banner.jpg" />
+      <div className="h-full d-flex">
+        <div className="Banner__container">
+          <Typography variant="h1">Buy your best books</Typography>
+          <Typography variant="h5" className="my-1 text-center" style={{ fontWeight: 500 }}>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
```

Comment:
You can remove this before merging the branch

Reply:
This is the desired text for banner. May be replace with a meaningful text. Thanks